os/arch/arm/src/armv8-m/up_exception.S

```diff
@@ -420,4 +451,13 @@ g_nestlevel:
 	.size   g_nestlevel, 4
 #endif
 
+	.bss
+	.global	g_bakirqstkalloc
+	.global	g_bakirqstkbase
+	.align	4
+g_bakirqstkalloc:
+	.skip	1024
```

Comment:
I think the size of nested irq stack can be configurable.
Please make config for this.

Reply:
Done